corehq/apps/app_manager/templates/app_manager/apps_base.html

```diff
@@ -71,6 +71,7 @@
     <script src="{% static 'app_manager/js/managed_app.js' %}"></script>
     <script src="{% static 'hqwebapp/js/bootstrap3/rollout_modal.js' %}"></script>
     <script src="{% static 'app_manager/js/section_changer.js' %}"></script>
+    <script src="{% static 'app_manager/js/custom_assertions.js' %}"></script>
```

Comment:
Is this now loaded on pages where it is not needed?

Reply:
Yes. That's typical for app manager (and HQ). It is possible to do something slightly more intricate to only import the library if the flag is on, particularly once this area is on webpack.